Assets/Css/style.css

```diff
@@ -357,6 +357,9 @@ html {
   position: absolute;
 }
 
+.video-gallery label:hover{
+  background-color: #49654c;
```

Comment:
both the text and the background colors are in the dark because the text seems to be disintegrating, You should make some changes and see whats looks best.

Reply:
Thanks for the suggestion! I have made some changes, hope that resolves the issue!